### PR TITLE
feat(dedicated.vcd): tracking vcd

### PR DESCRIPTION
**Pull request by dectotam**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/hpc-vcd-b0
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | Feat MANAGER-14798
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/vmware-cloud-director/controller.js [hunk 1]
```diff
@@ -20,6 +28,12 @@ export default class {
             ? 'dedicatedCloud_vmware_cloud_director_migration'
             : 'dedicatedCloud_vmware_cloud_director_validate_migration',
         );
+
+        if (this.vcdMigrationState.isDone) {
+          this.atInternet.trackPage({
+            name: `${this.trackingPrefix}vmware::vmware::banner-info::migrate_to_managed_vcd_success`,
+          });
+        }
```
Comment (darsene) [inline packages/manager/apps/dedicated/client/app/components/dedicated-cloud/dashboard/tiles/vmware-cloud-director/controller.js:36]: Not really sure it's well located. Migration will be done asynchronously and in a different timeline.
What's the condition to enable that tracking value ?

**Review by darsene — DISMISSED** [review]

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js [hunk 2]
```diff
@@ -98,8 +103,17 @@ export default /* @ngInject */ ($stateProvider) => {
...
+      goToVcdOrder: /* @ngInject */ (
+        $state,
+        atInternet,
+        trackingPrefix,
+      ) => () => {
+        atInternet.trackClick({
+          name: `${trackingPrefix}vmware::tile::button::migrate_to_managed_vcd::confirm`,
+          type: 'action',
+        });
+        $state.go('app.dedicatedCloud.details.dashboard.vcd-order');
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/dedicatedCloud-dashboard.routing.js:115]:
```suggestion
        return $state.go('app.dedicatedCloud.details.dashboard.vcd-order');
```

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/vmware-cloud-director/order/routes.js [hunk 3]
```diff
@@ -17,7 +17,17 @@ export default /* @ngInject */ ($stateProvider) =>
...
+      goBack: /* @ngInject */ (
+        goBackToDashboard,
+        atInternet,
+        trackingPrefix,
+      ) => () => {
+        atInternet.trackClick({
+          name: `${trackingPrefix}::vmware::tile::button::migrate_to_managed_vcd::close`,
+          type: 'action',
+        });
+        goBackToDashboard();
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/dashboard/vmware-cloud-director/order/routes.js:29]:
```suggestion
        return goBackToDashboard();
```

File: packages/manager/apps/dedicated/client/app/managedBaremetal/dashboard/dashboard.routing.js [hunk 4]
```diff
@@ -114,9 +119,17 @@ export default /* @ngInject */ ($stateProvider) => {
...
+      goToVcdOrder: /* @ngInject */ (
+        $state,
+        atInternet,
+        trackingPrefix,
+      ) => () => {
+        atInternet.trackClick({
+          name: `${trackingPrefix}vmware::tile::button::migrate_to_managed_vcd::confirm`,
+          type: 'action',
+        });
+        $state.go('app.managedBaremetal.details.dashboard.vcd-order');
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/managedBaremetal/dashboard/dashboard.routing.js:131]:
```suggestion
        return $state.go('app.managedBaremetal.details.dashboard.vcd-order');
```

File: packages/manager/apps/dedicated/client/app/managedBaremetal/dashboard/vmware-cloud-director/order/routes.js [hunk 5]
```diff
@@ -17,7 +17,17 @@ export default /* @ngInject */ ($stateProvider) =>
...
+      goBack: /* @ngInject */ (
+        goBackToDashboard,
+        atInternet,
+        trackingPrefix,
+      ) => () => {
+        atInternet.trackClick({
+          name: `${trackingPrefix}::vmware::tile::button::migrate_to_managed_vcd::close`,
+          type: 'action',
+        });
+        goBackToDashboard();
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/managedBaremetal/dashboard/vmware-cloud-director/order/routes.js:29]:
```suggestion
        return goBackToDashboard();
```

**Review by anooparveti — APPROVED** [review]

**Review by darsene — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Pull request merged**